packages/salesforce-adapter/test/filters/custom_objects_instances.test.ts

```diff
@@ -252,6 +253,7 @@ describe('Custom Object Instances filter', () => {
         saltoManagementFieldSettings: {
           defaultFieldName: MANAGED_BY_SALTO_FIELD_NAME,
         },
+        ...(omittedFields ? { omittedFields } : {}),
```

Comment:
```suggestion
       omittedFields,
```
Shouldn't it be like this..? I'm rather confused.

Reply:
If the array `omittedFields` is not empty then we want an object with a single entry with the key `"omittedFields"` and the value of the array `omittedFields`. But if the array is empty we don't want the object at all.

In other words, we want either
```
{
  ...
  omittedFields: [ ... ],
}
```
or
```
{
  ...
}
```
but never
```
{
  ...
  omittedFields: [],
}
```

Reply:
Why can't we have an empty array of **omittedFields** though?

Reply:
I just didn't want it in the test